pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "helm";
-  version = "3.7.0";
+  version = "3.7.1";
   gitCommit = "eeac83883cb4014fe60267ec6373570374ce770b";
```

Comment:
```suggestion
  gitCommit = "1d11fcb5d3f3bf00dbe6fe31b8412839a96b3dc4";
```

Reply:
Thanks. I'll squash this. Not sure why nixpkgs-update didn't do it...